kubernetes/.swagger-codegen/VERSION

```diff
@@ -0,0 +1 @@
+2.3.0-SNAPSHOT
```

Comment:
It's auto-generated, based on whatever the version [here](https://github.com/kubernetes-client/gen/blob/master/openapi/java.xml#L15) is pointing at. I locally updated that to point at the latest snapshot build. Trouble is, 2.3.0 isn't released yet, and updating that file to point at 2.3.0-SNAPSHOT requires you to checkout & build swagger-codegen locally to run the code generator. As far as I know, you can't point at a specific commit hash anyhow.